packages/manager/src/features/Support/SupportTickets/SupportTicketDrawer.tsx

```diff
@@ -447,7 +501,63 @@ export const SupportTicketDrawer: React.FC<CombinedProps> = (props) => {
     });
   };
 
-  const requirementsMet = description.length > 0 && summary.length > 0;
+  const renderDialogTitle = () => {
+    const general = 'Open a Support Ticket';
+    let dialogTitle: string;
+
+    switch (ticketType) {
+      case 'smtp':
+        dialogTitle = smtpDialogTitle;
+        break;
+      case 'general':
+      default:
+        dialogTitle = general;
+    }
+    return dialogTitle;
+  };
+
+  const renderHelperText = () => {
+    const general = (
+      <>
+        {`We love our customers, and we\u{2019}re here to help if you need us.
+        Please keep in mind that not all topics are within the scope of our support.
+        For overall system status, please see `}
+        <a
+          href="https://status.linode.com"
+          target="_blank"
+          aria-describedby="external-site"
+          rel="noopener noreferrer"
+        >
+          status.linode.com
+        </a>
+        .
+      </>
+    );
+    let helperText: string | React.ReactFragment;
+
+    switch (ticketType) {
+      case 'smtp':
+        helperText = smtpHelperText;
+        break;
+      case 'general':
+      default:
+        helperText = general;
+    }
+
+    return (
+      <Typography data-qa-support-ticket-helper-text>{helperText}</Typography>
```

Comment:
Suggestion: I think we can simplify this as below

```suggestion
   const isGeneralTicket = ticketType === 'general';

    return (
      <Typography data-qa-support-ticket-helper-text>
        {isGeneralTicket ? general : smtpHelperText}
      </Typography>
    );
```